packages/trip-details/src/index.tsx

```diff
@@ -169,26 +180,29 @@ export function TripDetails({
               transitFares={transitFares}
             />
           </summary>
-          {fareDetailsLayout ? (
-            // Show full ƒare details by leg
-            <FareLegTable layout={fareDetailsLayout} itinerary={itinerary} />
-          ) : (
-            // Just show the fares for each payment type
-            fareKeys.map(fareKey => {
-              // Don't show the default fare twice!
-              if (fareKey === defaultFare) {
-                return null;
-              }
-              return (
-                <TransitFare
-                  fareKey={fareKey}
-                  key={fareKey}
-                  fareKeyNameMap={fareKeyNameMap}
-                  transitFares={transitFares}
+          {fareDetailsLayout
+            ? // Show full ƒare details by leg
+              transitFares?.[defaultFare] && (
+                <FareLegTable
+                  layout={fareDetailsLayout}
+                  itinerary={itinerary}
```

Comment:
Sort props.

Reply:
Sorted in 26b50fef415e1dbf0339c25a520b56697861532b